### DISCUSSION TITLE
Create folders from albums?

### DISCUSSION THREAD

**Discussion by fmunteanu** [photoprism/photoprism#2684] [Q&A] [answered]
I have the `/originals` directory filled with 2000 images and no folders (just an example). My goal is create several albums which will implicitly generate folders with the pictures assigned to. For example, I will create a `Screenshots` album and move inside 100 pictures. The outcome would be to see an `/originals/screenshots` containing the 100 pictures. Is this feasible through UI or some CLI form?

**Comment by lastzero** [comment, marked as answer]
https://docs.photoprism.app/getting-started/faq/#can-i-use-photoprism-to-sort-files-into-a-configurable-folder-structure

**Reply by fmunteanu** [reply]
Thanks, in other words, I have to handle the folders in /originals myself, no other way around.

**Reply by lastzero** [reply]
Not without writing a bit of code or using external tools at this time.

**Reply by graciousgrey** [reply]
Yes, unless you want to have your files ordered by year/month, in this case you could use our import functionality: https://docs.photoprism.app/user-guide/library/#importing-files. 

Moving files within originals using the PhotoPrism Web UI  might be implemented at a later point.
